cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -24,9 +24,29 @@ Cask Data Application Platform Release Notes
    :depth: 2
 
 
-`Release 3.0.1 <http://docs.cask.co/cdap/3.0.1/index.html>`__
+`Release 3.0.3 <http://docs.cask.co/cdap/3.0.3/index.html>`__
```

Comment:
Can we drop the changes to this file? I don't think that in the 3.1 release we should mention side branch release changes.